patches/chrome-android-BUILD.gn.patch

```diff
@@ -15,7 +15,7 @@ index dd59ffa4ffb9bd12d28e10aa64dc7dcfdabc0849..dd6cff83b144e5e7a572b97d883e4db6
      "//third_party/android_deps:com_android_support_preference_v7_java",
      "//third_party/android_deps:com_android_support_recyclerview_v7_java",
    ]
-+  deps += ["//brave/build/android:android_brave_strings_grd"]
++  deps += brave_chrome_app_java_resources_deps resource_dirs += brave_chrome_app_java_resources_dirs
```

Comment:
Just confirming: assignments can be separated with only whitespace?

Reply:
yep

Reply:
I did this same thing here https://github.com/brave/brave-core/pull/5064/files#diff-592bd965aabfe4ab456e5454131e530eR59

Reply:
OK gotcha!